Bombay hospitality Ltd. operates a franchise model for producing exotic Norwegian dinners throughout New England. The operating cost for a franchise in a week (W) is given by the equation W = $1,000 + $5X, where X represents the number of units produced in a week. Recent feedback from restaurant owners suggests that this cost model may no longer be accurate, as their observed weekly operating costs are higher.
Objective:
To investigate the restaurant owners' claim about the increase in weekly operating costs using hypothesis testing.


In [1]:
#import the libraries
import pandas as pd
import scipy
import numpy as np
from scipy import stats

# Given

Sample mean: 
𝑥bar = 3050

Theoretical mean: 
mu0 = 1000 + 5 × 600 = 4000

SD used: 
𝜎 = 5×25
=125


Sample size: 

n=25


We use the z-test ( known true population σ).



In [4]:
import math
from scipy import stats   #

In [16]:
# Given values
xbar = 3050              # sample mean
mu0 = 1000 + 5*600       # theoretical mean = 4000
sigma = 5 * 25           # population std dev = 125
n = 25                   # sample size


In [22]:
# Standard error and Z-statistic
se = sigma / math.sqrt(n)
z_stat = (xbar - mu0) / se
se

25.0

Hypotheses statement:

Null Hypothesis (H₀):
𝜇=4000

(No increase in cost; model is accurate)

Alternative Hypothesis (H₁):

𝜇>4000
(Costs are higher than what the model suggests)

In [19]:
# Critical value for one-tailed test
alpha = 0.05
z_crit = stats.norm.ppf(1 - alpha)   # one-tailed (right tail)

In [20]:
# Decision
print("\nTest Statistic (Z):", z_stat)
print("Critical Value (Z_crit):", z_crit)



Test Statistic (Z): -38.0
Critical Value (Z_crit): 1.6448536269514722


In [21]:
if z_stat > z_crit:
    print("Decision: Reject H0 → Costs are significantly higher.")
else:
    print("Decision: Fail to Reject H0 → No strong evidence of higher costs.")

Decision: Fail to Reject H0 → No strong evidence of higher costs.


# Conclusion

Calculated test statistic: z = -38.0

Critical value (one-tailed, α = 0.05, df=24): t = 1.6449

Since z < t_crit, we fail to reject H₀.

Conclusion: There is no statistical evidence to support the restaurant owners’ claim that weekly costs are higher than the model prediction. 
In fact, the sample mean (3050) is much lower than the expected cost (4000).